ingestion-beam/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM maven:3-jdk-8 AS base
+WORKDIR /app/ingestion-beam
+
+COPY LICENSE /app/LICENSE
+COPY pom.xml /app/pom.xml
+COPY ingestion-beam/pom.xml /app/ingestion-beam/pom.xml
+COPY checkstyle /app/checkstyle
+COPY ingestion-beam/checkstyle /app/ingestion-beam/checkstyle
+COPY ingestion-core/src /app/ingestion-core/src
+COPY ingestion-beam/src /app/ingestion-beam/src
+COPY ingestion-beam/bin /app/ingestion-beam/bin
```

Comment:
It may be more efficient to implement a .dockerignore file to handle excluding the content we don't need before it even gets transferred to the Docker context. Then we could simplify these COPY statements.

Reply:
I'll think about this as an optimization in a second pass. I'm working through getting the integration workflow running inside of the container, but this would be desirable. There's >1GB of context that's sent to docker everytime I build, likely leftovers from development that I've done. I do like this pattern of being explicit about the dependencies, but it's pretty verbose too.